.github/workflows/pr.yml

```diff
@@ -2,38 +2,25 @@ name: PR
 
 on:
   pull_request:
-    branches: [ master ]
+    branches:
+      - master
+      - feature/sumida-upgrade
```

Comment:
What is the reason for this change?

Reply:
@dweng0 change for Quarto migration

Reply:
There shouldn't be any need to specify branch-specific behaviour in Github workflows. The original version should be restored in this PR or develop branch.